js/sign.js

```diff
@@ -0,0 +1,52 @@
+const errorMessageEmail = document.querySelector('.enter-email');
+const errorMessageEmail2 = document.querySelector('.wrong-email');
```

Comment:
-Email1, -Email2와 같은 이름보다는 조금 더 의미 있게 변수명을 지어볼 수 있을 것 같아요~

Reply:
변수명에서 email이 앞에 오는게 더 적합할 것 같아요!

```suggestion
const emailErrorMessage = document.querySelector('.enter-email');
const emailErrorMessage2 = document.querySelector('.wrong-email');
```

Reply:
클래스명도 조금 더 디테일하게 수정해볼 수 있을 것 같네요!